index.html

```diff
@@ -5689,6 +5696,9 @@ <h3>HTML Attribute State and Property Mappings</h3>
               </td>
               <td class="aria">
                 <a class="core-mapping" href="#focus_state_event_table">See Focus States and Events Table</a>
+                <p>
+                  If set to an element with no corresponding role, then map to the <a class="core-mapping" href="#role-map-group">`group`</a> role.
```

Comment:
Similar notes to above... and one more question:
Did you want to say 'If set to a value >= "0" then...' ?
(because this includes "-1" as-is - not sure if you wanted to include that)

```suggestion
                  If set on an element with no role, then map to the <a class="core-mapping" href="#role-map-group">`group`</a> role.
```